### ISSUE TITLE
Doesn't work as they put JS

### ISSUE THREAD

**Issue by greenamit** [sevensins/Wallbase-Downloader#1]
Hi,

I downloaded and configured the script as required. It doesn't work anymore.
I then tried to debug it and found that they insert a JS that only populate the real img url upon loading.
It looks something like that:

<div id="bigwall" class="right">

    <script type="text/javascript">document.write('<img alt="Sparta signs funny warning caution stick figures awesomeness  / 1920x1200 Wallpaper" src="'+B('aHR0cDovL2ltZzMuaW1hZ2VzaGFjay51cy9pbWczLzc0MTEvd2FsbHBhcGVyNzgyMzUyLmpwZw==')+'" />');</script>

</div>

Where src="+B" calls a JS function.

So basically, the following command  in the script returns nothing (and nothing get downloaded):
cat $number | egrep -o “http:.*(gif|png|jpg)” | egrep “wallbase2|imageshack.us|ovh.net” | wget –keep-session-cookies –load-cookies=cookies.txt –referer=http://wallbase.cc/wallpaper/$number -i -

**Comment by macearl** [comment]
Fixed it.

The Javascript doesnt really matter, they just encrypted the urls with base64.

The Script now uses base64 -d to decode those urls

**Comment by greenamit** [comment]
Works!

Thanks a lot!

**Comment by macearl** [comment]
No problem. Thx for creating the issue, wouldn't have noticed it for probably a month or so
